startinstance/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@eformservice/startinstance",
+  "servicename": "startinstance",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint --ext .ts,.tsx ./",
+    "test": "cross-env HOTRELOAD=no DEBUG=bpmn-engine:123* ts-mocha -p tsconfig.json ./**/*.roxtest.ts --exit --recursive --timeout 40000 --require regenerator-runtime",
+    "copyproduction": "cross-var copyfiles package.json production && cd ./production && npm install --production && cd .. && cross-var copyfiles -u 1 production/node_modules/** $npm_package_servicename && rimraf production/",
```

Comment:
Please use the new format here for the copyfiles command so that the zip is correctly built afterwards.
Just copy the copyproduction script from another service from master.